presto-parser/src/main/java/com/facebook/presto/sql/tree/ColumnDefinition.java

```diff
@@ -29,24 +29,25 @@
     private final String type;
     private final List<Property> properties;
     private final Optional<String> comment;
+    private final boolean nullable;
```

Comment:
Let's move this to be after `type`, here and everywhere else in the class, so that it's in the same order as in the SQL definition (also that ordering seems more logical / readable)